pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -357,6 +366,33 @@ func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kube
 		return err
 	}
 
+	// Set sysctls if requested, using the CNI tuning plugin
```

Comment:
Split the whole rkt support into another PR on-top of this: https://github.com/kubernetes/kubernetes/pull/31080